akka-stream-tests/src/test/scala/akka/stream/impl/StreamLayoutSpec.scala

```diff
@@ -124,21 +126,23 @@ class StreamLayoutSpec extends AkkaSpec {
     }
 
     "not fail materialization when building a large graph with simple computation" when {
+      // Seen tests run in 9-10 seconds, these test cases are heavy on the GC
```

Comment:
yeah, it seems to be very heavy on GC. 
```
2357.321: [GC (Allocation Failure) [PSYoungGen: 593259K->29833K(618496K)] 1925032K->1361613K(2016768K), 0.1259008 secs] [Times: user=1.12 sys=0.02, real=0.13 secs] 
2360.995: [GC (Allocation Failure) --[PSYoungGen: 618121K->618121K(618496K)] 1949901K->2016389K(2016768K), 1.2152092 secs] [Times: user=14.67 sys=7.55, real=1.21 secs] 
2362.210: [Full GC (Ergonomics) [PSYoungGen: 618121K->69251K(618496K)] [ParOldGen: 1398268K->1398243K(1398272K)] 2016389K->1467494K(2016768K), [Metaspace: 372791K->372774K(1368064K)], 3.6699223 secs] [Times: user=89.71 sys=0.77, real=3.67 secs] 
[info]   - when starting from a Source *** FAILED *** (9 seconds, 283 milliseconds)
[info]     A timeout occurred waiting for a future to complete. Queried 20 times, sleeping 15 milliseconds between each query. (StreamLayoutSpec.scala:134)
```

Perhaps we should give it even more time to complete?

What's not so nice with such garbage generating tests is that GC can easily spill over to other tests (later tests or tests running in parallel).